packages/cli/docs/modules/ROOT/pages/index.adoc

```diff
@@ -0,0 +1,40 @@
+= Command Line Interface (CLI)
+
+*Develop, deploy and operate upgradeable smart contract projects*. Support for Ethereum and every other EVM and eWASM-powered blockchain.
```

Comment:
```suggestion
*Develop, deploy and operate upgradeable smart contract projects* on Ethereum using the OpenZeppelin CLI.
```

Reply:
I'd rather not repeat 'OpenZeppelin CLI' again. Do you want to remove the bit about other EVM blockchains? It's on the [current readme](https://github.com/OpenZeppelin/openzeppelin-sdk#openzeppelin-sdk)

Reply:
Hmm let's drop the eWASM bit, keep the EVM one.